core/src/Revolution/Sources/modMediaSource.php

```diff
@@ -545,21 +545,25 @@ public function getObjectsInContainer($path)
 
 
     /**
-     * @param $path
+     * @param string $path
+     * @param bool $isNew
      *
      * @return array|bool
      */
-    public function getMetaData($path)
+    public function getMetaData($path, $isNew = false)
```

Comment:
Changing the signature of a public method of a non-final class is a technical breaking change (as derivative classes could have extended the method). There are no derivatives of this method in core and it was only added in 3.0, so perhaps we can choose to ignore introducing a breaking change at this point in the release, or we could just re-evaluate why we're adding this and go with the easiest approach of just lowering the error level on the message?

Reply:
I'd argue that's completely a technicality, as a default value is given for the new property and it's _only_ used to determine whether the thrown error is to be expected or not.

Reply:
I'm with Mark on this one. I'd rather see the error level reduced here. The isNew parameter just doesn't even feel right on the getMetaData() method.

Reply:
I agree, It's not right, nor is how this method is leaned on to figure out the writability of a file. Given that I see _this_ PR's fix as a temporary bridge to the real and hopefully much more complete fix I'm working on, it's not a big deal changing this the way you're requesting. Will update later today...

Reply:
OK, the requested change was made. All - please be aware that this PR _does not_ fix the problems with saving non-media source files.